wlauto/workloads/appshare/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -78,9 +76,9 @@ private void setupGooglePhotos() throws Exception {
 
     private void sendToGmail(String recipient) throws Exception {
         String gID = gmail.packageID;
-
+        
         shareUsingApp("Gmail", "gmail");
-
+        
```

Comment:
Are they actually extra spaces? I think it's just showing that I deleted a blank line and then re-added it to keep the original format?